projects/go-lib/src/lib/components/go-select/go-select.component.spec.ts

```diff
@@ -206,4 +226,32 @@ describe("GoSelectComponent", () => {
       fixture.debugElement.queryAll(By.css("go-form-errors"))?.length
     ).toBe(1);
   });
+
```

Comment:
Are these tests required? I dont think they are anyway related to the change being made.

Reply:
Code removed. Hence resolving.